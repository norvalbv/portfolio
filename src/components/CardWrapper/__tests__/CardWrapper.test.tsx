import React from 'react';
import renderer from 'react-test-renderer';

import Accordion from '..';

describe('<Accordion />', () => {
  test('should render correctly', () => {
    const tree = renderer.create(<Accordion title="Test Title">Test Content</Accordion>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
