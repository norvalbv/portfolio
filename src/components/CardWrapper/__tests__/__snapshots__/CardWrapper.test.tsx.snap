// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CardWrapper /> should render correctly with all props 1`] = `
<div
  className="relative top-[-4rem] z-10 mx-auto min-h-screen bg-red-500"
  id="testId"
>
  <div>
    <header
      className="flex items-center justify-between"
    >
      <section
        className="flex w-full flex-col gap-1"
      >
        <h2
          className="text-blue-500 mb-10 text-4xl font-semibold underline underline-offset-8 md:mb-20 md:text-6xl"
        >
          j
          h
          t
          z
          x
          v
          e
          x
          r
          t
        </h2>
        <h4
          className="text-body text-neutral-N300 truncate"
          data-testid="Test subtitle Subtitle"
        >
          Test subtitle
        </h4>
        <p
          className="text-body text-neutral-N00 truncate"
          data-testid="Test description Description"
        >
          Test description
        </p>
      </section>
    </header>
    <div>
      Child
    </div>
  </div>
</div>
`;

exports[`<CardWrapper /> should render correctly with mandatory props 1`] = `
<div
  className="relative top-[-4rem] z-10 mx-auto min-h-screen grid w-10/12 min-w-[20rem] place-items-center"
>
  <div>
    <header
      className="flex items-center justify-between"
    >
      <section
        className="flex w-full flex-col gap-1"
      >
        <h2
          className="mb-10 text-4xl font-semibold underline underline-offset-8 md:mb-20 md:text-6xl"
        >
          x
          v
          u
          l
          s
          x
          f
          p
          h
          z
        </h2>
      </section>
    </header>
    <div>
      Child
    </div>
  </div>
</div>
`;
